packages/user_packages/iotcc_mqtt.py

```diff
@@ -97,6 +100,9 @@ def run(self, registry):
                     self.iotcc.set_properties(self.iotcc_edge_system, {"key1": "value1", "key2": "value2"})
                     break
                 except Exception:
+                    # In the third attempt if get exception raise it
+                    if attempts == 3:
```

Comment:
Please extract 3 as a constant/variable NUMBER_OF_ATTEMPTS instead of specifying it at multiple places..

Reply:
Fixed